README.md

```diff
@@ -74,6 +74,12 @@ The callbacks for all options are called whenever a set of arguments is parsed.
 If an option isn't provided in the args, its callback is passed the default
 value, or `null` if no default value is set.
 
+Options can be mandatory (required) with the flag `mandatory` set to `true`.
```

Comment:
These semantics match well with `required` from the language. I think we should use that term everywhere rather than the term "mandatory" which gets clarified as "required".


```suggestion
If an option is `required` but not provided, the parser throws an [`ArgParserException`][ArgParserException].
```